<launch>
  <!-- LIDAR Driver -->
  <node name="rplidarNode" pkg="rplidar_ros" type="rplidarNode" output="screen">
    <param name="serial_port" type="string" value="/dev/ttyUSB0"/>
    <param name="serial_baudrate" type="int" value="115200"/>
    <param name="frame_id" type="string" value="laser"/>
    <param name="inverted" type="bool" value="false"/>
    <param name="angle_compensate" type="bool" value="true"/>
  </node>
  
  <!-- LIDAR Processor -->
  <node name="lidar_processor" pkg="lidar_navigation" type="lidar_processor.py" output="screen">
    <rosparam command="load" file="$(find lidar_navigation)/config/lidar_params.yaml"/>
  </node>
</launch>